GEOSldas_App/preprocess_ldas_routines.F90

```diff
@@ -2155,7 +2164,10 @@ subroutine optimize_latlon(fname_tilefile, N_proc_string, optimized_file, run_di
           n2 = k*IMGLOB
           face_land(k) = sum(landPosition(n1:n2)) 
           face(k) = nint(1.0*face_land(k)/total_land * N_proc)
```

Comment:
@weiyuan-jiang: Is this safe without parenthesis or the use of `real()`?  The variables `face_land`, `total_land`, and `N_proc` are all integers. Does the multiplication with 1.0 always force all multiplications and divisions to be real?  Or is there any possibility that an integer division kicks in before the conversion to real?

Reply:
> @weiyuan-jiang: Is this safe without parenthesis or the use of `real()`? The variables `face_land`, `total_land`, and `N_proc` are all integers. Does the multiplication with 1.0 always force all multiplications and divisions to be real? Or is there any possibility that an integer division kicks in before the conversion to real?

That 1.0 promotes all the integer type to real